seqr/management/commands/check_bam_cram_paths.py

```diff
@@ -49,3 +53,16 @@ def handle(self, *args, **options):
             logger.info('{} files not found:'.format(sum(missing_counter.values())))
             for project_name, c in sorted(missing_counter.items(), key=lambda t: -t[1]):
                 logger.info('   {} in {}'.format(c, project_name))
+
+            # post to slack
+            safe_post_to_slack(
+                SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
+                'Found {} broken bam/cram path(s)'.format(sum(missing_counter.values())),
+            )
+            slack_message = ""
+            for project_name, missing_paths_list in project_name_to_missing_paths.items():
+                slack_message += "In project {}\n".format(project_name)
+                slack_message += "\n".join([
+                    "  {}   {}".format(individual_id, path) for individual_id, path in missing_paths_list
+                ])
+                safe_post_to_slack(SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL, slack_message)
```

Comment:
Is there a reason to do this as separate slacks for each project instead of one big post? My concern is sometimes we get errors when we send too many slack messages in a row where a couple fail to go through because it gets overloaded, and if we only send one message for the job that won't be an issue